src/route/router.ts

```diff
@@ -475,6 +477,62 @@ router.post('/video/pending/approved', approvePendingReviewVideoController) // 
 // https://localhost:9999/video/pending/approved
 // cookie: uid, token
 
+// ========== 新的审核接口（需要 developer 角色，使用 uuid 和 token） ==========
+
+router.get('/review/video/pending', getPendingReviewVideoListController) // 获取待审核视频列表（需要 developer 角色）
+// https://localhost:9999/review/video/pending?num=20&offset=0
+// cookie: uuid, token
+
+router.get('/review/comment/pending', getPendingReviewCommentListController) // 获取待审核评论列表（需要 developer 角色）
+// https://localhost:9999/review/comment/pending?num=20&offset=0
+// cookie: uuid, token
+
+router.get('/review/danmaku/pending', getPendingReviewDanmakuListController) // 获取待审核弹幕列表（需要 developer 角色）
```

Comment:
```suggestion
router.get('/review/danmaku/pending', getPendingReviewDanmakuListController) // 获取待审核弹幕列表
```
通过 RBAC 限制访问，不用标明接口权限了